### PR TITLE
Make App Py3-ready

### PR DESCRIPTION
**Pull request by ekoval**

### CODE REVIEW

File: healthcheck/__init__.py [hunk 1]
```diff
@@ -3,6 +3,6 @@
 __url__ = 'https://github.com/yola/healthcheck'
 
 
-from healthcheck import (HealthChecker, HealthCheck, ListHealthCheck,
+from .healthcheck import (HealthChecker, HealthCheck, ListHealthCheck,
```
Comment (snitch) [inline healthcheck/__init__.py]: F401 '.healthcheck.ListHealthCheck' imported but unused
F401 '.healthcheck.HealthCheck' imported but unused
F401 '.healthcheck.FilesExistHealthCheck' imported but unused
F401 '.healthcheck.HealthChecker' imported but unused
F401 '.healthcheck.DjangoDBsHealthCheck' imported but unused
F401 '.healthcheck.FilesDontExistHealthCheck' imported but unused
I101 Imported names are in the wrong order. Should be DjangoDBsHealthCheck, FilesDontExistHealthCheck, FilesExistHealthCheck, HealthCheck, HealthChecker, ListHealthCheck

File: healthcheck/healthcheck.py [hunk 2]
```diff
@@ -20,22 +16,21 @@ def __init__(self, is_critical=True, check_id=None):
...
         self._ok = None
         self._details = {}
 
-    @abc.abstractmethod
     def run(self):
         """If you create your own HealthCheck, it should implement .run()
         method, which sets self._ok and self._details properties."""
-        pass
+        raise ValueError('You must override "run" method for check %s.' %
+                             (self.__class__.__name__,))
```
Comment (snitch) [inline healthcheck/healthcheck.py]: E127 continuation line over-indented for visual indent

File: tests/test_healthcheck.py [hunk 3]
```diff
@@ -22,15 +22,14 @@ class TestHealthCheck(TestCase):
...
-        self.assertRaisesRegexp(TypeError, 'Can\'t instantiate', HealthCheck,
-                                dict(check_id='111'))
-
     def test_cant_create_healthcheck_without_run_method(self):
         class MyCheck(HealthCheck):
-            pass
-
-        self.assertRaisesRegexp(TypeError, 'Can\'t instantiate', MyCheck)
+            check_id = 'check_id'
+        print(MyCheck().__class__.__name__)
```
Comment (snitch) [inline tests/test_healthcheck.py]: T003 print function found.

File: healthcheck/__init__.py [hunk 4]
```diff
@@ -2,7 +2,7 @@
 __version__ = '0.0.7'
 __url__ = 'https://github.com/yola/healthcheck'
 
-
-from healthcheck import (HealthChecker, HealthCheck, ListHealthCheck,
+ROOT_URLCONF = ''
+from .healthcheck import (HealthChecker, HealthCheck, ListHealthCheck,
```
Comment (snitch) [inline healthcheck/__init__.py]: E402 module level import not at top of file
F401 '.healthcheck.ListHealthCheck' imported but unused
F401 '.healthcheck.HealthCheck' imported but unused
F401 '.healthcheck.FilesExistHealthCheck' imported but unused
F401 '.healthcheck.HealthChecker' imported but unused
F401 '.healthcheck.DjangoDBsHealthCheck' imported but unused
F401 '.healthcheck.FilesDontExistHealthCheck' imported but unused
I101 Imported names are in the wrong order. Should be DjangoDBsHealthCheck, FilesDontExistHealthCheck, FilesExistHealthCheck, HealthCheck, HealthChecker, ListHealthCheck

File: healthcheck/contrib/django/status_endpoint/tests/__init__.py [hunk 5]
```diff
@@ -0,0 +1,8 @@
+import os
+
+import django
+
+
+os.environ.setdefault(
+    'DJANGO_SETTINGS_MODULE', 'healthcheck.contrib.django.status_endpoint.settings')
```
Comment (snitch) [inline healthcheck/contrib/django/status_endpoint/tests/__init__.py]: E501 line too long (84 > 79 characters)

File: healthcheck/contrib/django/status_endpoint/tests/test_views.py [hunk 6]
```diff
@@ -3,20 +3,15 @@
 
 os.environ['DJANGO_SETTINGS_MODULE'] = 'test_settings'
 
-from django.conf import settings
 from django.core.urlresolvers import reverse
 from django.db import connections
-from django.test import TestCase
+from django.test.testcases import TestCase
```
Comment (snitch) [inline healthcheck/contrib/django/status_endpoint/tests/test_views.py]: E402 module level import not at top of file

File: tests/__init__.py [hunk 7]
```diff
@@ -0,0 +1,8 @@
+import os
+
+import django
+
+
+os.environ.setdefault(
+    'DJANGO_SETTINGS_MODULE', 'healthcheck.contrib.django.status_endpoint.settings')
```
Comment (snitch) [inline tests/__init__.py]: E501 line too long (84 > 79 characters)

File: tests/test_healthcheck.py [hunk 8]
```diff
@@ -1,8 +1,10 @@
 # -*- coding: utf-8 -*-
-from tempfile import NamedTemporaryFile
 import errno
-from unittest2 import TestCase
+import os
```
Comment (snitch) [inline tests/test_healthcheck.py]: F401 'os' imported but unused

File: tests/test_healthcheck.py [hunk 9]
```diff
@@ -1,8 +1,10 @@
...
+from tempfile import NamedTemporaryFile
 
+from unittest2 import TestCase
+from django.test.utils import override_settings
```
Comment (snitch) [inline tests/test_healthcheck.py]: F401 'django.test.utils.override_settings' imported but unused

File: healthcheck/__init__.py [hunk 10]
```diff
@@ -2,7 +2,6 @@
 __version__ = '0.0.7'
 __url__ = 'https://github.com/yola/healthcheck'
 
-
-from healthcheck import (HealthChecker, HealthCheck, ListHealthCheck,
-                         DjangoDBsHealthCheck, FilesExistHealthCheck,
-                         FilesDontExistHealthCheck)
+from .healthcheck import (DjangoDBsHealthCheck, FilesDontExistHealthCheck,
```
Comment (snitch) [inline healthcheck/__init__.py]: F401 '.healthcheck.ListHealthCheck' imported but unused
F401 '.healthcheck.DjangoDBsHealthCheck' imported but unused
F401 '.healthcheck.FilesExistHealthCheck' imported but unused
F401 '.healthcheck.HealthChecker' imported but unused
F401 '.healthcheck.HealthCheck' imported but unused
F401 '.healthcheck.FilesDontExistHealthCheck' imported but unused
I101 Imported names are in the wrong order. Should be DjangoDBsHealthCheck, FilesDontExistHealthCheck, FilesExistHealthCheck, HealthCheck, HealthChecker, ListHealthCheck

File: healthcheck/contrib/django/status_endpoint/tests/__init__.py [hunk 11]
```diff
@@ -0,0 +1,6 @@
+import os
+
+import django
```
Comment (snitch) [inline healthcheck/contrib/django/status_endpoint/tests/__init__.py]: F401 'django' imported but unused

File: healthcheck/__init__.py [hunk 12]
```diff
@@ -2,7 +2,6 @@
 __version__ = '0.0.7'
 __url__ = 'https://github.com/yola/healthcheck'
 
-
-from healthcheck import (HealthChecker, HealthCheck, ListHealthCheck,
-                         DjangoDBsHealthCheck, FilesExistHealthCheck,
-                         FilesDontExistHealthCheck)
+from .checks import (DjangoDBsHealthCheck, FilesDontExistHealthCheck,
```
Comment (snitch) [inline healthcheck/__init__.py]: F401 '.checks.ListHealthCheck' imported but unused
F401 '.checks.DjangoDBsHealthCheck' imported but unused
F401 '.checks.FilesExistHealthCheck' imported but unused
F401 '.checks.HealthChecker' imported but unused
F401 '.checks.HealthCheck' imported but unused
F401 '.checks.FilesDontExistHealthCheck' imported but unused
I101 Imported names are in the wrong order. Should be DjangoDBsHealthCheck, FilesDontExistHealthCheck, FilesExistHealthCheck, HealthCheck, HealthChecker, ListHealthCheck

Comment (RayeN) [reply healthcheck/__init__.py]: Use noqa?  And now you can do absolute import here.

File: healthcheck/checks.py [hunk 13]
```diff
@@ -1,13 +1,9 @@
 from __future__ import absolute_import
```
Comment (RayeN) [inline healthcheck/checks.py]: We don't need this anymore

File: tests/test_healthcheck.py [hunk 14]
```diff
@@ -1,8 +1,8 @@
 # -*- coding: utf-8 -*-
-from tempfile import NamedTemporaryFile
 import errno
-from unittest2 import TestCase
+from tempfile import NamedTemporaryFile
 
+from unittest2 import TestCase
```
Comment (RayeN) [inline tests/test_healthcheck.py]: Do we still need this for Python 2.7?

File: tests/test_checks.py [hunk 15]
```diff
@@ -1,12 +1,13 @@
...
 import errno
-from unittest2 import TestCase
+from tempfile import NamedTemporaryFile
+from unittest import TestCase
 
 from mock import patch
 
-from healthcheck import (HealthChecker, HealthCheck, ListHealthCheck,
-                         FilesExistHealthCheck, FilesDontExistHealthCheck)
+from healthcheck.checks import (
```
Comment (snitch) [inline tests/test_checks.py]: I101 Imported names are in the wrong order. Should be FilesDontExistHealthCheck, FilesExistHealthCheck, HealthCheck, HealthChecker, ListHealthCheck

File: healthcheck/__init__.py [hunk 16]
```diff
@@ -1,8 +1,7 @@
 __doc__ = 'Health Checker for Yola Services'
-__version__ = '0.0.7'
+__version__ = '0.1.0'
 __url__ = 'https://github.com/yola/healthcheck'
 
-
-from healthcheck import (HealthChecker, HealthCheck, ListHealthCheck,
-                         DjangoDBsHealthCheck, FilesExistHealthCheck,
-                         FilesDontExistHealthCheck)
+from .checks import (DjangoDBsHealthCheck, FilesDontExistHealthCheck,
```
Comment (snitch) [inline healthcheck/__init__.py]: F401 '.checks.ListHealthCheck' imported but unused
F401 '.checks.DjangoDBsHealthCheck' imported but unused
F401 '.checks.FilesExistHealthCheck' imported but unused
F401 '.checks.HealthChecker' imported but unused
F401 '.checks.HealthCheck' imported but unused
F401 '.checks.FilesDontExistHealthCheck' imported but unused
I101 Imported names are in the wrong order. Should be DjangoDBsHealthCheck, FilesDontExistHealthCheck, FilesExistHealthCheck, HealthCheck, HealthChecker, ListHealthCheck

File: healthcheck/__init__.py [hunk 17]
```diff
@@ -1,8 +1,7 @@
...
+                    FilesExistHealthCheck, HealthChecker, HealthCheck,
```
Comment (snitch) [inline healthcheck/__init__.py]: E128 continuation line under-indented for visual indent

File: healthcheck/__init__.py [hunk 18]
```diff
@@ -1,8 +1,7 @@
...
+                    ListHealthCheck)
```
Comment (snitch) [inline healthcheck/__init__.py]: E128 continuation line under-indented for visual indent

File: healthcheck/__init__.py (hunk 16 above)
Comment (snitch) [inline healthcheck/__init__.py]: F401 '.checks.ListHealthCheck' imported but unused
F401 '.checks.DjangoDBsHealthCheck' imported but unused
F401 '.checks.FilesExistHealthCheck' imported but unused
F401 '.checks.HealthChecker' imported but unused
F401 '.checks.HealthCheck' imported but unused
F401 '.checks.FilesDontExistHealthCheck' imported but unused
I101 Imported names are in the wrong order. Should be DjangoDBsHealthCheck, FilesDontExistHealthCheck, FilesExistHealthCheck, HealthCheck, HealthChecker, ListHealthCheck

File: requirements.txt [hunk 19]
```diff
@@ -1,7 +1,6 @@
 coverage == 3.7.1
-django>=1.4,< 1.7
+django>=1.7,<1.12
```
Comment (RayeN) [inline requirements.txt]: Let's use specific version libs for testing.

File: tests/test_checks.py (hunk 15 above)
Comment (RayeN) [reply tests/test_checks.py]: I think Snitch is right here.

File: healthcheck/__init__.py (hunk 16 above)
Comment (snitch) [inline healthcheck/__init__.py]: F401 '.checks.ListHealthCheck' imported but unused
F401 '.checks.DjangoDBsHealthCheck' imported but unused
F401 '.checks.FilesExistHealthCheck' imported but unused
F401 '.checks.HealthChecker' imported but unused
F401 '.checks.HealthCheck' imported but unused
F401 '.checks.FilesDontExistHealthCheck' imported but unused
I101 Imported names are in the wrong order. Should be DjangoDBsHealthCheck, FilesDontExistHealthCheck, FilesExistHealthCheck, HealthCheck, HealthChecker, ListHealthCheck

**Pull request merged**